backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/materialDispo/distributionAgg.feature

```diff
@@ -0,0 +1,260 @@
+# ###############################################################################################################################################
+# ###############################################################################################################################################
+# ###############################################################################################################################################
+# ###############################################################################################################################################
+  @from:cucumber
+  Scenario: Aggregate by orderLineId
+    When set sys config boolean value false for sys config DDOrderAggregation.header.bySalesOrderId
+    And set sys config boolean value true for sys config DDOrderAggregation.line.bySalesOrderLineId
+    And metasfresh contains C_Orders:
+      | Identifier | IsSOTrx | C_BPartner_ID | DateOrdered | PreparationDate      | M_Warehouse_ID |
+      | SO1        | true    | bpartner_1    | 2022-07-20  | 2022-07-17T00:00:00Z | targetWH       |
+      | SO2        | true    | bpartner_1    | 2022-07-20  | 2022-07-17T00:00:00Z | targetWH       |
+    And metasfresh contains C_OrderLines:
+      | Identifier | C_Order_ID | M_Product_ID | QtyEntered |
+      | ol_1       | SO1        | p_1          | 2          |
+      | ol_2       | SO1        | p_1          | 3          |
+      | ol_3       | SO2        | p_1          | 4          |
+      | ol_4       | SO2        | p_1          | 5          |
+    And the order identified by SO1 is completed
+    And wait until de.metas.material rabbitMQ queue is empty or throw exception after 5 minutes
+    And the order identified by SO2 is completed
+    And wait until de.metas.material rabbitMQ queue is empty or throw exception after 5 minutes
+
+    And after not more than 30s, following DD_Order_Candidates are found
+      | Identifier | M_Product_ID | M_Warehouse_From_ID | M_WarehouseTo_ID | Qty   | Processed |
+      | c1         | p_1          | sourceWH            | targetWH         | 2 PCE | N         |
+      | c2         | p_1          | sourceWH            | targetWH         | 3 PCE | N         |
+      | c3         | p_1          | sourceWH            | targetWH         | 4 PCE | N         |
+      | c4         | p_1          | sourceWH            | targetWH         | 5 PCE | N         |
+
+    And the following DD_Order_Candidates are enqueued for generating DD_Orders
+      | DD_Order_Candidate_ID |
+      | c1                    |
+      | c2                    |
+      | c3                    |
+      | c4                    |
+
+    Then metasfresh contains DD_Orders:
+      | Identifier | C_BPartner_ID.Identifier | M_Warehouse_ID.From.Identifier | M_Warehouse_ID.To.Identifier | M_Warehouse_ID.Transit.Identifier | C_DocType_ID.Name    |
```

Comment:
consider getting rid of `.Identifier` suffix if possible

Reply:
removed.